src/library-authoring/data/api.ts

```diff
@@ -56,6 +56,10 @@ export const getXBlockFieldsApiUrl = (usageKey: string) => `${getApiBaseUrl()}/a
   * Get the URL for the xblock OLX API
   */
 export const getXBlockOLXApiUrl = (usageKey: string) => `${getLibraryBlockMetadataUrl(usageKey)}olx/`;
+/**
+* Get the URL for the xblock Publish API
+*/
```

Comment:
Nit: The indentation of all your JSDoc comments is wrong - [the stars should all be aligned in a column](https://jsdoc.app/about-getting-started).
```suggestion
 * Get the URL for the xblock Publish API
 */
```